docs/openapi.yml

```diff
@@ -351,9 +357,12 @@ paths:
                   status:
                     type: string
                   data:
+                    type: object
                     properties:
                       cohort:
-                        $ref: '#/components/schemas/Cohort'
+                        properties:
+                          id:
+                            $ref: '#/components/schemas/Cohort'
```

Comment:
I'm not confident in this. an `id` property should not be an entire `Cohort` schema